packages/@azure/servicebus/data-plane/test/README.md

```diff
@@ -52,9 +51,46 @@
     TOPIC_FILTER_NAME=topic-filter
     TOPIC_FILTER_SUBSCRIPTION_NAME=topic-filter-subscription
     TOPIC_FILTER_DEFAULT_SUBSCRIPTION_NAME=topic-filter-default-subscription
-
     ```
 
+## Setup
+
+Go through the following setup inorder to delete and create the required servicebus-entities(just before running each test) and for authenticating to the servicebus using `AadTokenCredentials` instead of `ConnectionString`.
```

Comment:
> Go through the following setup in order to delete and create the required servicebus-entities(just before running each test). This would need authenticating to Service Bus using `AadTokenCredentials` instead of connection string. The below setup is also needed to run the tests that specifically uses `AadTokenCredentials` to authenticate.

With the above update, you can skip the "Note" that follows